src/home/home.scss

```diff
@@ -172,7 +172,7 @@
     &.fairLaunch {
       display: grid;
       grid-template-columns: auto 1fr;
-      column-gap: 10%;
+      column-gap: 5em;
```

Comment:
We are not using em nor rem.  And that you find are not supposed to be there.  We definitely don't want to add more.  Can you please convert to px?